medcat/utils/saving/envsnapshot.py

```diff
@@ -0,0 +1,60 @@
+from typing import List, Dict, Any, Set
+
+import os
+import re
+import pkg_resources
+import platform
+
+
+ENV_SNAPSHOT_FILE_NAME = "environment_snapshot.json"
+SETUP_PY_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "..", "setup.py"))
```

Comment:
Both of these module level vars are not needed anymore

Reply:
Removed